client/src/components/Monument/RelatedMonuments/RelatedMonuments.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import "./RelatedMonuments.scss";
+import Spinner from "../../Spinner/Spinner";
+import SearchResult from "../../Search/SearchResult/SearchResult";
+import Collapse from "react-bootstrap/Collapse";
+
+/**
+ * Presentational Component for Related Monuments
+ */
+export default class RelatedMonuments extends React.Component {
```

Comment:
I think you could combine RelatedMonuments and NearbyMonuments, and just pass the title text in as a prop

Reply:
Unless there's a difference between them that I missed

Reply:
Yeah, I can do that.

Reply:
Done.